cluster/juju/layers/kubernetes-worker/metadata.yaml

```diff
@@ -27,4 +23,4 @@ resources:
   kubernetes:
     type: file
     filename: kubernetes.tar.gz
-    description: "A tarball packaged release of the kubernetes bins."
+    description: "An archive of kubernetes binaries for the worker."
```

Comment:
I dont see the master messaging updated for it's package. It doesn't have to be in this PR but we should also normalize those messages that these are specially packaged releases for the charms and not the upstream release bin.

This is in addendum to sam's question's earlier today which indicated we had not been clear on that message.